.gitignore

```diff
@@ -1,152 +1,6 @@
 
-# Created by https://www.gitignore.io/api/vim,code,emacs,python,pycharm,virtualenv,sublimetext
```

Comment:
FYI: Better to move user-specific (e.g., editor stuff) ignore rules to a non-version controlled space. See https://git-scm.com/docs/gitignore and https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file. Note that you can do so globally (for all your Git repos) and project-specific (by putting it in `.git/info/exclude`). The version-controlled `.gitignore` file should ideally only contain the stuff that needs to be ignored by _all_ developers. It's of course a minor thing, but good to know